site/content/en/_index.html

```diff
@@ -4,22 +4,22 @@
 
 +++
 
-{{< blocks/cover title="Host, run and scale dedicated game servers on Kubernetes" image_anchor="top" height="full" color="orange" >}}
+{{< blocks/cover title="" image_anchor="top" height="full" color="white" >}}
```

Comment:
I'm not sure how, but was thinking to remove the `td-cover-block--height-full` css class from this section in the redered html looks nicer (so it doesn't take up the full page)

Reply:
changed it td-cover-block--height-min and it much better !